src/modules/article/article.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Article } from '../database/entities/article.entity';
+import { ArticleDto } from '../database/dto/article.dto';
+
+@Injectable()
+export class ArticleService {
+  constructor(
+    @InjectRepository(Article)
+    private readonly articleRepository: Repository<Article>,
+  ) {}
+
+  async getAll(): Promise<ArticleDto[]> {
+    return this.articleRepository.find();
+  }
+
+  async getById(id: string): Promise<ArticleDto> {
+    return this.articleRepository.findOne(id);
+  }
+
+  async getArticlesByUserId(userId: string): Promise<ArticleDto[]> {
+    return this.articleRepository.find({
+      where: { userId: `${userId}` },
+    });
+  }
+
+  async createOne(article: ArticleDto): Promise<ArticleDto> {
+    const newArticle = await this.articleRepository.create(article); // crypto in Node
+    await this.articleRepository.save(newArticle);
+    return newArticle;
+  }
+
+  async update(id: string, article: ArticleDto): Promise<ArticleDto> {
+    await this.articleRepository.update(id, article);
+    return this.articleRepository.findOne(id);
```

Comment:
update with returning

Reply:
I changed that way
 return this.articleRepository.save({ id, article });